### ISSUE TITLE
Difficulty floating in water

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11434]
**Preferred term label:**
Difficulty floating in water

**Synonyms**

**Definition (free text, please give PubMed ID)**
The body is more dense than typical for other people

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
autosomal dominant osteopetrosis-4 (OPTA4) https://omim.org/entry/621449
Presumably other osteopetroses too but not mentioned in OMIM.

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001460
- label: Difficulty floating in water
- parents: [Constitutional symptom (HP:0025142)](https://hpo.jax.org/app/browse/term/HP:0025142)
- pmid: PMID:21455762
- def: A phenomenon whereby an individual struggles to float, cannot swim despite adequate training, or simply sinks. This phenomenon may be observed in individuals with conditions such as osteopetrosis that are characterized by substantially increased bone mineral density (which implies reduced bouyancy).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11434
- synonyms:
